www/source/blog/2018-06-13-base-plans-refresh.html.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Base Plans Refresh is Coming to Habitat Core Plans!
+date: 2018-06-13
+author: Nell Shamrell-Harrington
+tags: core plans
+category: update
+classes: body-article
+---
+
+Greetings, fellow Habicats!
+
+We will be refreshing the base plans of Core Plans on Tuesday, June 19th.  Let's go over what this means for us and, more important, what it means for you.
+
+## Base Plans 101
+
+Currently, there are about 500 Habitat plans under the Core Origin on Builder.
+
+These include everything from services like Postgresql to binaries like go to very low level system libraries, all of which you can easily pull into your own Habitat artifacts.
+
+### What's a Base Plan?
+
+The Base Plans are a group of plans (mostly low level systems libraries) that:
+* are used by nearly every other plan (or a dependency of that plan) on Builder
+* are used to build Habitat itself
+* need to be built in a certain order.
+
+You can see [the full list of Base Plans](https://github.com/habitat-sh/core-plans/blob/master/CODEOWNERS) in the Core Plans CODEOWNERS file.
+
+Any time one of these plans is updated (especially ones that are nearly universally used, like [glibc](https://en.wikipedia.org/wiki/GNU_C_Library) and [gcc](https://gcc.gnu.org/), all other base plans which depend on it need to be built in a certain order, and then all plans that depend (or depend on a plan that depends) on those base plans also need to be rebuilt. We call this the "rebuild the world" scenario.
+
+### Why are you refreshing them?
+
+Many of our current base plans are on very old versions - i.e. the current version of gcc on Builder is 5.2.0, when the most recent stable release of gcc is 8.1.0).  It is vital to update them - and consequently every plan that depends on them - to more recent stable versions.
+
+Many of them also must be updated simultaneously - if we update gcc to 8.1 and it requires a later version of glibc than we current have, we must also update glibc.
+
+### What else does it affect?
+
+We are simultaneously refreshing nearly every [core plan](https://github.com/habitat-sh/core-plans/) at the same time as we are refreshing the base plans.  This is because the majority (if not all) of the core plans depend on at least one of the base plans in some fashion (or, at the very least, a base plan is in their dependency chain).
```

Comment:
> core plan

Perhaps core Plan?

Reply:
…for that matter it might be worth scanning for `plan` -> `Plan`

Reply:
Excellent point!

Reply:
Fixed!